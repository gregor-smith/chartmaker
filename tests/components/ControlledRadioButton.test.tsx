test.todo('renders labelled radio button')


test.todo('change event calls onCheck prop')
