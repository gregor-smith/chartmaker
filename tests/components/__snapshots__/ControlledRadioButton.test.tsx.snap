// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders labelled radio button 1`] = `
.emotion-1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-1:not(:first-of-type) {
  margin-left: 0.5rem;
}

.emotion-1:not(:last-of-type) {
  margin-right: 0.5rem;
}

.emotion-0 {
  margin-right: 0.25rem;
}

<div
  id="test-render-container"
>
  <div
    class="emotion-1"
  >
    <input
      checked=""
      class="emotion-0"
      id="test-radio-button"
      type="radio"
    />
    <div
      class="mock-label"
    >
      Target: test-radio-button
      Test children
    </div>
  </div>
</div>
`;
